inst/scripts/objective-batch.R

```diff
@@ -0,0 +1,31 @@
+#### Test script for objective function analysis
+
+library('gcamland')
+library('doParallel')
+
+## nodefile: File containing the list of nodes
+## nproc:    Total number of cores available ($SLURM_NTASKS)
+## N:        Number of iterations to run
+## outdir:   Output directory
+## skip:     Number of iterations to skip (i.e., if resuming from a previous run)
+## logdir:   Directory for logs
+run_ens_obj_analysis <- function(nodefile, nproc, N, outdir, skip=0, logdir=NULL)
+{
+    print(getwd())
+    nodes <- readr::read_lines(nodefile)
+    ncore <- ceiling(nproc / length(nodes)) # cores per node
+    nodes <- rep(nodes, ncore)
+
+    cl <- makeCluster(nodes, outfile="")
+    registerDoParallel(cl)
+
+    print(system.time(run_ensemble(N, outdir, skip, logparallel=logdir)))
+    stopCluster(cl)
+
+    # TODO: update to include objective function analysis
+    # run_objective can be run in parallel and explictly saved.
+    # grand_table_objective and MAP_objective need to work on
+    # the results of run_objective for all models of interest.
```

Comment:
Did you want to do this in this PR or is there a reason to wait?

Reply:
@kvcalvin this is the last comment I'm addressing.

I've updated the code to do the `run_objective` and `grand_table` steps .

`minimize_objective` can look at different metrics over different crops, so I implemented the version that does RMS for all crops. I have it save those results and the `grand_table` results so that other combinations can be performed offline.

Do you have alternative thoughts on the best way to do that?

There's also a decent chance that I did not implement the saving correctly. I believe that   `scenObjects` gets created in `run_ensemble` before the parallel part, and `scenObjects` is what is returned by `run_ensemble` and then gets passed to the other functions. (aka doesn't need parallel). So I think my calls to the functions are ok, but I'm not sure if I'm giving things the right ID (`suffix`)  when saving. 
 
I'm running the final local package checks before pushing all edits now